cvat/apps/engine/views.py

```diff
@@ -579,87 +584,44 @@ def upload_finished(self, request: ExtendedRequest):
-            - GET /api/requests/<rq_id> to check export status,
-                where rq_id is request id returned on initializing request'
+            - `POST /api/projects/<project_id>/dataset/export?save_images=False` to initiate exporting process
+            - `GET /api/requests/<rq_id>` to check export status,
+                where `rq_id` is request id returned on initializing request
+            - `GET result_url` to download a prepared file with annotations,
+                where `result_url` can be found in the response on checking status request
         """),
-        parameters=[
-            OpenApiParameter('format', description='Desired output format name\n'
-                'You can get the list of supported formats at:\n/server/annotation/formats',
-                location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=True),
-            OpenApiParameter('filename', description='Desired output file name',
-                location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=False),
-            OpenApiParameter('action', description='Used to start downloading process locally after annotation file has been created',
-                location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=False, enum=['download']),
-            OpenApiParameter('location', description='Where need to save downloaded dataset',
-                location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=False,
-                enum=Location.list()),
-            OpenApiParameter('cloud_storage_id', description='Storage id',
-                location=OpenApiParameter.QUERY, type=OpenApiTypes.INT, required=False),
-            OpenApiParameter('use_default_location', description='Use the location that was configured in project to export annotation',
-                location=OpenApiParameter.QUERY, type=OpenApiTypes.BOOL, required=False,
-                default=True, deprecated=True),
-        ],
         responses={
-            '200': OpenApiResponse(PolymorphicProxySerializer(
-                component_name='AnnotationsRead',
-                serializers=[LabeledDataSerializer, OpenApiTypes.BINARY],
-                resource_type_field_name=None
-            ), description='Download of file started'),
-            '201': OpenApiResponse(description='Annotations file is ready to download'),
-            '202': OpenApiResponse(description='Dump of annotations has been started'),
-            '401': OpenApiResponse(description='Format is not specified'),
-            '405': OpenApiResponse(description='Format is not available'),
-        })
-    @action(detail=True, methods=['GET'],
-        serializer_class=LabeledDataSerializer,
-        csrf_workaround_is_needed=csrf_workaround_is_needed_for_export)
+            '410': OpenApiResponse(description="API endpoint is no longer handles exporting process"),
```

Comment:
Do we even need to keep this endpoint in the schema if it doesn't do anything useful? I can see the value of keeping the endpoint itself, so that we can return a useful status to any existing client, but what purpose does keeping it in the schema serve?

Reply:
Probbaly not to break SDK?

Reply:
But you deleted the query parameters from the schema, which would break it anyway.

(Also, just to clarify: what SDK usage scenario are you talking about? Someone using the low-level API with the current version of the SDK to interact with an old version of CVAT?)

Reply:
> Also, just to clarify: what SDK usage scenario are you talking about? 

New SDK version + new server version. But okay, I've deleted these endpoints from the server schema.

Reply:
now it's done

Reply:
> New SDK version + new server version.

When used with a new server version, the SDK functions corresponding to the removed endpoints would not work anyway.